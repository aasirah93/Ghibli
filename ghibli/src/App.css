
/* ol {list-style-type: none;
}; */

.box{
  border: 5px red solid;
  border-width: 3px;
  border-style: thick;
  border-radius : 10px;
  width: 300px;
  float: left;
  margin: 0 20px 20px 0;
  padding: 20px;
  font-size: 20px;
};

.font-size{
  font-size: 10px;
}




