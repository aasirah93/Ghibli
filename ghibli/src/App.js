import React from 'react';
import './App.css';
import Header from './components/Header.js';
import Ghibli from './components/Ghibli.js';


function App() {
//   const [items, setItems] = useState([]);
//   const[isLoading, setIsLoading] = useState(true);

//   const fetchData = () => {fetch('https://ghibliapi.herokuapp.com/films')
//   .then(response => response.json())
//   .then(
//   (data) => {
  
//   setItems(data);
//   setIsLoading(false);

// },

// )}

// useEffect(() => {
//   fetchData()
// }, [])
// console.log(items)

//  return (

//    isLoading?  (<h1>Loading...</h1>)  :
return(

   <div>
     <Header/>
     <Ghibli/>
     {/* <ol style={{ listStyleType: "none" }}>
     {items && items.map(item => (
     <li key = {item.id}>
    <div class="box">
     <h1> {item.title}</h1>
     <h2> Director {item.director}</h2>
     <h3 class="font-size"> Released {item.release_date}</h3>
     <p>{item.description}</p>
     </div>
     </li>)
     )}
     </ol> */}
  </div>
  //  )
)
}
export default App;



