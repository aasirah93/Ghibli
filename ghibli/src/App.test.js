import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

it('retrieves data', () => {
  const fetchData = () => {fetch('https://ghibliapi.herokuapp.com/films')};
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
