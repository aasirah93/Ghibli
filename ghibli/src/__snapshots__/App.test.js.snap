// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h1>
  Loading...
</h1>
`;

exports[`retrieves data 1`] = `
<h1>
  Loading...
</h1>
`;
